app/src/main/java/org/p2p/wallet/newsend/ui/NewSendPresenter.kt

```diff
@@ -205,6 +205,10 @@ class NewSendPresenter(
         }
     }
 
+    private suspend fun getNonZeroUserTokens(): List<Token.Active> {
+        return tokenServiceCoordinator.getUserTokens().filterNot { it.isZero }
```

Comment:
which is better?
it.isZero or it.total.isZero?

Reply:
idk 🙂